qunetsim/objects/connections/quantum_connection.py

```diff
@@ -0,0 +1,47 @@
+from qunetsim.objects.connections.channel_models.binary_erasure import BinaryErasure
+from qunetsim.objects.connections.connection import Connection
+
+class QuantumConnection(object):
```

Comment:
Similarly here, should inherit from `connection`

Reply:
Then you can remove the setters and getters that overlap between the two objects since they'll be inherited

Reply:
Done